src/Coverage.jl

```diff
@@ -136,28 +137,31 @@ module Coverage
             @info "Coverage.process_cov: processing $file"
             coverage = CovCount[]
             for line in eachline(file)
-                # Columns 1:9 contain the coverage count
-                cov_segment = line[1:9]
                 # If coverage is NA, there will be a dash
-                push!(coverage, cov_segment[9] == '-' ? nothing : parse(Int, cov_segment))
+                if line[9] == '-'
+                    push!(coverage, nothing)
+                else
+                    # Columns 1:9 contain the coverage count
+                    push!(coverage, parse(Int, line[1:9]))
+                end
```

Comment:
This part of the changes seems to be purely cosmetically, or am I missing something?

Reply:
The rule has now become: if the coverage is already empty (`-`), or there is a line exclusion tag, `exclude`. Otherwise, it parses columns 1 to 9.